vendor/data/codelists/BEIS/aid_type.yml

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Does this file duplicate content in `vendor/data/codelists/IATI/2_03/activity/aid_type.yml`? If so, how will future developers know how to keep these two copies up to date? Or is there some way we can share content between them?

Reply:
The way we usually work when adapting codelists from IATI is copying the IATI codes and making them BEIS codelists. Part of the aim of this piece of work was to better codify some of the rules, rather than having to override certain behaviours with constants etc. I think any change that is made to any codelists in future will be done slowly and deliberately, so there's minimal risk of this getting missed / confused.

Reply:
Okay, that makes sense 👍